src/index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<!-- This sample is extended from the GMP Maps JavaScript API tutorial:
+  https://developers.google.com/maps/documentation/javascript/tutorial -->
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Cloud Tasks Pizza Map</title>
+    <style>
+      /* Set map height explicitly to define the size of the div element that contains the map. */
+      #map {
+        height: 100%;
+      }
+      html, body {
+        height: 100%;
+        margin: 0;
+        padding: 0;
+      }
+    </style>
+  </head>
+  <body>
+    <div id="map"></div>
+    <script>
```

Comment:
I definitely considered it, but it would require another Express route and splitting ~120 lines into 20 and 100 doesn't have more pros than cons. Happy to discuss more though if there's disagreement.